.ci/setenvconfig

```diff
@@ -122,6 +122,7 @@ TEST_LICENSE = /go/src/github.com/elastic/cloud-on-k8s/.ci/test-license.json
 MONITORING_SECRETS = /go/src/github.com/elastic/cloud-on-k8s/.ci/monitoring-secrets.json
 
 OPERATOR_IMAGE = $JKS_PARAM_OPERATOR_IMAGE
+TEST_TIMEOUT = 10m
```

Comment:
why?

Reply:
Because when I ran the test manually I had to increase the timeout (with Kind everything is running on the same host, it can be very slow, there's a lot of I/O, and some beats spend some random times in a `CrashloopBackoff` state).
I suspect the test is almost reaching that limit and may hit it at some point. Increasing the timeout is harmless and will prevent to have some false positive. (I guess the timeout has been already increased to 10m for eks/aks and ocp for the same reason).